Source/stores.cpp

```diff
@@ -973,33 +952,35 @@ void StoreConfirm()
 	AddSTextVal(8, item._iIvalue);
 	PrintStoreItem(item, 9, itemColor);
 
+	const char *prompt;

```

Comment:
```suggestion
	string_view prompt;
```

A fatal exception is thrown if the default case is hit so this is overly defensive, but this ensures the default `string_view` constructor is used giving a 0 length string instead of potentially calling AddSText with a garbage pointer.

Reply:
Nice, we should get rid of as much `char *` as possible.